### DISCUSSION TITLE
Anywhere in GSA that requires or allows inputting an IP address, please allow inputting of a domain name instead

### DISCUSSION THREAD

**Discussion by mixels** [gameserverapp/Platform#944] [Suggestions]
We use DDNS to emulate a static IP address since we self-host our servers and our internet service provider will not provide a static IP. It's already a big hassle to change IP addresses when things change on services that don't support domain name resolution. If you stored the value entered for the host and parsed it to see if it matches IPv4 format, then resolve hostname if not, it would be a tremendous help in this regard and a big product differentiator. For whatever reason, very few other services do this, and it's a big win for self-hosting communities.

Please consider!

**Comment by mvaessen** [comment]
Hi, Thanks for your suggestion.

Not all parts of GSA are able to work with DNS, apart from the extra lookups and issues it may introduce. Therefor am I going to reject this suggestion. People who don't have access to a (somewhat) static IP should consider renting a machine from a datacenter.
